tasks/seedManagement.js

```diff
@@ -0,0 +1,35 @@
+const { task } = require("hardhat/config");
+const { SeedArguments } = require("../test/args/seed-test-args.json");
+
+// Arguments for creating a Seed are taken from "../test/args/seed-test-args.json"
+task("createSeed", "Creates a Seed directly, bypassing Gnosis safe").setAction(
+  async (_, hre) => {
+    console.log("Creating Seed...");
+    const seedFactoryInstance = await hre.ethers.getContract("SeedFactory");
+    const tx = await seedFactoryInstance.deploySeed(
+      SeedArguments.BENEFICIARY,
+      SeedArguments.ADMIN,
+      [SeedArguments.TD2D, SeedArguments.TUSDC],
+      [SeedArguments.softCap, SeedArguments.hardCap],
+      SeedArguments.price,
+      SeedArguments.startTime,
+      SeedArguments.endTime,
+      [SeedArguments.vestingDuration, SeedArguments.vestingCliff],
+      SeedArguments.isPermissioned,
+      SeedArguments.fee,
+      SeedArguments.metadata
+    );
+    console.log(`Seed created through tx ${tx.hash}`);
+    await tx.wait();
+    console.log("Seed deployed");
+  }
+);
+
+task("changeOwner", "changes owner of SeedFactory")
+  .addParam("address", "new owner address", undefined)
+  .setAction(async ({ address }, { ethers }) => {
+    console.log(`changing owner of SeedFactory to ${address}`);
+    const seedFactoryInstance = await ethers.getContract("SeedFactory");
```

Comment:
optional: add try/catch for encountering unexpected errors (for the whole function body)